Tests/test_integration.py

```diff
@@ -186,6 +186,30 @@ def __disable_integrations_instances(client, module_instances):
             print_error(pformat(res))
 
 
+def __enable_integrations_instances(client, module_instances):
```

Comment:
add docstring

Reply:
You don't think this function is fairly self-explanatory?